test/e2e/auth/audit.go

```diff
@@ -23,21 +23,30 @@ import (
 	"strings"
 
 	apiv1 "k8s.io/api/core/v1"
+	extensions "k8s.io/api/extensions/v1beta1"
+	apiextensionsv1beta1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
+	"k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
+	"k8s.io/apiextensions-apiserver/test/integration/testserver"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apiserver/pkg/apis/audit/v1beta1"
 	"k8s.io/kubernetes/test/e2e/framework"
+	imageutils "k8s.io/kubernetes/test/utils/image"
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 )
 
+const (
+	watchTimeout = 1
+	user         = "kubecfg"
```

Comment:
is this name important? Can it get a less generic name (it's in some package with much other stuff) ?

Reply:
I would say yes. For example there are multiple get requests in the audit log, some of them generated by other components during pod (or deployment creation). So I'm using user name to filter requests that have been actually send from the test code. Unfortunately I haven't found a way how we can change kubecfg name to something else.

Reply:
sounds all fine. Would rename it to `auditTestUser = "kubecfg"`. Similar for `watchTimeout`. Then it lgtm.

Reply:
Done.

Reply:
same for `watchTimeout` please. Then I can tag the PR.

Reply:
Done. Thanks for the review!